File: SDKs/apidapp.com/2019-02-14T164701Z/typescript/src/sdk/models/operations/geterc20.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



export class GetErc20Response extends SpeakeasyBase {
  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  empty?: Record<string, any>;

  @SpeakeasyMetadata()
  statusCode: number;
}

File: SDKs/apidapp.com/2019-02-14T164701Z/python/src/sdk/models/operations/get_erc20.py
import dataclasses
from typing import Any,Optional


@dataclasses.dataclass
class GetErc20Response:
    content_type: str = dataclasses.field()
    status_code: int = dataclasses.field()
    empty: Optional[dict[str, Any]] = dataclasses.field(default=None)
    
